metricbeat/docs/modules/redis/key.asciidoc

```diff
@@ -0,0 +1,23 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[metricbeat-metricset-redis-key]]
+=== Redis key metricset
+
+beta[]
```

Comment:
Should we go GA with this directly?

Reply:
Ok, let's go for GA, I forgot to add the beta warning in the metricset in any case :smile: